product_docs/docs/livecompare/2/bdr_support.mdx

```diff
@@ -5,23 +5,20 @@ originalFilePath: bdr_support.md
 
 ---
 
-You can use LiveCompare against EDB Postgres Distributed (PGD, formerly known as BDR) nodes as well as non-PGD nodes.
+LiveCompare is an analysis tool. You can use it with EDB Postgres Distributed (PGD, formerly known as BDR) nodes and non-PGD nodes.
+When you set  `logical_replication_mode = bdr`, LiveCompare  considers that all databases being compared belong to the same PGD cluster. You can specify node names as connections and replication sets to filter tables.
```

Comment:
Comment from Jayashree for a developer:  Is this a parameter? where are we actually setting this. specifying that will be helpful